src/QMCDrivers/VMC/VMCBatched.h

```diff
@@ -24,17 +25,46 @@ namespace qmcplusplus
  */
 class VMCBatched : public QMCDriverNew
 {
+public:
+  using PosType = QMCTraits::PosType;
+  /** To avoid 10's of arguments to runVMCStep
+   */
+  struct StateForThread
```

Comment:
This seems like it has everything except for thread-specific state?   It needs a better name.   QMCInput?  QMCSimulation?